include/stdexec/execution.hpp

```diff
@@ -101,6 +123,31 @@ namespace stdexec {
   }
   using __get_completion_signatures::get_completion_signatures_t;
 
+  /////////////////////////////////////////////////////////////////////////////
+  // get_attrs
+  namespace __attrs {
+
+    struct __empty_attrs {
+      using __t = __empty_attrs;
+      using __id = __empty_attrs;
+    };
+
+    struct get_attrs_t {
+      template <class _Sender>
+        requires tag_invocable<get_attrs_t, const _Sender&>
+        constexpr auto operator()(const _Sender& __sender) const
+          noexcept(nothrow_tag_invocable<get_attrs_t, const _Sender&>)
+          -> tag_invoke_result_t<get_attrs_t, const _Sender&> {
+          static_assert(queryable<tag_invoke_result_t<get_attrs_t, const _Sender&>>);
+          return tag_invoke(*this, __sender);
+        }
```

Comment:
I'm making a change to the spec of `get_attrs` so that if <code><i>is-awaitable</i>&lt;S&gt;</code> is true, it returns <code><i>empty-attrs</i></code>. Would be good to make an analogous change here.